scss/components/_transitions.scss

```diff
@@ -0,0 +1,62 @@
+.transition-area {
+  background-color: #ccc;
+  position: relative;
+
+  &::after {
+    content: '';
+    display: block;
+    padding-top: 20%;
+  }
+
+  &:hover {
+    .transition-child {
+      left: calc(100% - 50px);
+      top: 0;
+    }
+  }
+}
+
+.transition-child {
+  width: 60px;
+  height: 60px;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.transition-child--1 {
+  transition-duration: 2s;
+  transition-timing-function: cubic-bezier(0.35, 0.27, 0.62, 0.93);
+}
+
+.transition-child--2 {
+  padding-top: 60px;
```

Comment:
suggestion: no need to use padding for all these transition childs, they are all position: absolute already, you can simply use the `top` here

Reply:
Cum spunea Olguta - eu oleaca "tupesc"...
Nu inteleg ce aveti in vedere...